feat(blog): add bpmn-js@9 release blog post

marstamm opened the pull request:
<!--

Thanks for creating this pull request!

Please make sure to link the issue you are closing as "Closes #issueNr". 
This helps us to understand the context of this PR.

-->

nikku reviewed (commented):
Great post.

And kindof hillarious how short it is for the time we've spent on this topic. But all things are in there.

I've added a bunch of changes on top via https://github.com/bpmn-io/bpmn.io/pull/123/commits/d9c1f49761d4055f5f051d3e812fc831be2923da. Please have a look (improved screen capture, small text changes).

Specifically I more prominently speak about _drill down_ as this is something I believe a lot of people are aware of.

marstamm commented:
Thank you for the feedback! Makes sense to me. I really like how it turned out.

I updated the download links and squashed the commits. I think we are ready to publish this
